core/src/main/scala/com/salesforce/op/evaluators/Evaluators.scala

```diff
@@ -54,10 +54,11 @@ object Evaluators {
 
     /*
      * Brier Score for the prediction
+     *
+     * @param numOfBins number of bins to produce
      */
-    def brierScore(): OpBinScoreEvaluator =
-      new OpBinScoreEvaluator(
-        name = BinaryClassEvalMetrics.brierScore, isLargerBetter = true)
+    def brierScore(numOfBins: Int = 100): OpBinScoreEvaluator =
```

Comment:
brierScore is independent of numBins.

Reply:
Then perhaps we should not even expose it. Why is it 100 by default?

Reply:
numBins are needed to compute other stats in OpBinScoreEvaluator like averageScore per bin.
As we are interested only in brierScore here, we don't need to pass numBins.

Reply:
got it. removed.